game-core/src/main/java/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -908,11 +908,13 @@ private void resetPlayers() {
     players = new ArrayList<>();
   }
 
-  private void setPlayerAttachmentName(final String name) throws GameParseException {
-    if (name == null) {
-      playerAttachmentName = null;
+  private void setPlayerAttachmentName(final String playerAttachmentName)
+      throws GameParseException {
+    if (playerAttachmentName == null) {
+      this.playerAttachmentName = null;
```

Comment:
Codacy found an issue: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6168405)